CHARTER.md

```diff
@@ -8,17 +8,18 @@ This Technical Charter sets forth the responsibilities and procedures for techni
 
 #### 1. Mission and Scope of the Technical Initiative
 
-- a. The mission of the Technical Initiative is to provide open source developers with best practices recommendations, and with an easy way to learn and apply them.
+The Best Practices for OSS Developers working group is dedicated to 
+- helping identify and curating an accessible inventory of best practices; 
+- raising awareness; 
+- and education of secure code best practices for open source developers.
 
-- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
+#### 2. Technical initiative roles
 
-#### 2. Technical Steering Committee
+- a. The primary points of contact are:
+  - Working group lead: [Christopher "CRob" Robinson](https://github.com/RedHatCRob)
+  - Working group co-lead: [Xavier René-Corail](https://github.com/xcorail)
 
-- a. The Technical Steering Committee (the "TSC") will be responsible for all oversight of the Technical Initiative. 
-
-- b. The TSC voting members are initially the Technical Initiative’s Maintainers. The Maintainers will be documented in the Technical Initiative repository. The TSC is responsible for determining the future process for defining voting members of the TSC, and any such alternative approach will also be documented appropriately.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
-
-- c. The Technical Initiative generally will involve Collaborators and Contributors. The TSC may adopt or modify additional roles so long as the roles are documented in the Technical Initiative’s repository. Unless otherwise documented: 
+- b. The Technical Initiative generally will involve Collaborators and Contributors. The working group may adopt or modify additional roles so long as the roles are documented in the Technical Initiative’s repository. Unless otherwise documented: 
```

Comment:
Based on some of the Voting stuff below, as well as the duties listed for them, we should probably create a Team for the Collaborators then use that not only for setting merge permissions on the repo but also as a list for determining voting rights and quorum. If that's the way we decide to go, then we should probably define it in the doc somehow.